indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -108,37 +107,36 @@ def _process_args(self):
         self.editable_type = EditableType[request.view_args['type']]
 
     def _process_GET(self):
-        return jsonify(editing_settings.get(self.event, self.editable_type.name + '_review_conditions').items())
+        query = (
+            EditingReviewCondition.query.with_parent(self.event)
+            .filter_by(type=self.editable_type)
+            .order_by(EditingReviewCondition.id)
+            .all()
+        )
+        return jsonify([[cond.id, sorted([ft.id for ft in cond.file_types])] for cond in query])
```

Comment:
`sorted(...)` instead of `sorted([...])` (internally sorted() already creates a list when passing a generator)